File: Java/1534. Count Good Triplets.java
class Solution {
    public int countGoodTriplets(int[] arr, int a, int b, int c) {
        int ans = 0, N = arr.length;

        for (int i = 0; i < N; i++) {
            for (int j = i + 1; j < N; j++) {
                for (int k = j + 1; k < N; k++) {
                    if (Math.abs(arr[i] - arr[j]) <= a && 
                        Math.abs(arr[j] - arr[k]) <= b && 
                        Math.abs(arr[i] - arr[k]) <= c) {
                            ans++;
                        }
                }
            }
        }

        return ans;
    }
}

File: Python3/1534. Count Good Triplets.py
class Solution:
    def countGoodTriplets(self, arr: List[int], a: int, b: int, c: int) -> int:
        N = len(arr)
        ans = 0

        for i in range(N):
            for j in range(i + 1, N):
                for k in range(j + 1, N):
                    if  abs(arr[i] - arr[j]) <= a and \
                        abs(arr[j] - arr[k]) <= b and \
                        abs(arr[i] - arr[k]) <= c:
                        ans += 1

        return ans
